aleph-client/src/session.rs

```diff
@@ -58,15 +58,22 @@ pub fn set_keys(connection: &Connection, new_keys: Keys, status: XtStatus) {
     send_xt(connection, xt, Some("set_keys"), status);
 }
 
+/// Get the number of the current session.
+/// Panics for the 0th session.
 pub fn get_current(connection: &Connection) -> u32 {
+        try_get_current(connection).unwrap()
```

Comment:
actually using here unwrap_or(0) and removing the function `try_get_current` makes more sense.

Reply:
Agreed.